.github/workflows/publish-cloud.yml

```diff
@@ -21,18 +21,24 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v3
-      - name: get configuration
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+      - name: Get Configuration
         id: configuration
         uses: antifree/json-to-variables@v1.0.1
         with: 
             filename: './.github/workflows/config.json'
             prefix: config
       - name: Test permissions
         uses: ./.github/workflows/permissions
-      - name: Install Node.js 14.x
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY: ${{ env.config_deployment_repository }}
```

Comment:
Should be the current repo owner, currently it's unless, because the check is:
`if [[ "${{ inputs.DEPLOYMENT_REPOSITORY }}" != "${{ env.config_deployment_repository }}/"* ]];`
It will always be true.

Reply:
done